### DISCUSSION TITLE
Chatbot with Python

### DISCUSSION THREAD

**Discussion by MartinRojo1** [community/community#75615] [Programming Help]
### Body

I need to create a chatbot on WhatsApp, I know it is possible to use Meta API in the cloud, but what other technology would be useful?

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by elderguardian** [comment]
https://www.geeksforgeeks.org/building-whatsapp-bot-on-python/

**Comment by kur0neko** [comment]
I think Meta is a common approach.  I found there are several tools you should try https://influencermarketinghub.com/whatsapp-chatbot-tools/
